appservice/thirdpartynotices.txt

```diff
@@ -36,3 +36,61 @@ OTHER DEALINGS IN THE SOFTWARE.
 END OF node-archiver NOTICES AND INFORMATION
 =========================================
 
+2. simple-git version 1.80.1 (https://github.com/steveukx/git-js)
```

Comment:
This isn't quite the format we're looking for. Take a look at this example: https://github.com/Microsoft/vscode-azurefunctions/blob/master/thirdpartynotices.txt